pkg/client-sdk/example/covenantless/alice_to_bob.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	log.Infof("alice offchain balance: %d", aliceBalance.OffchainBalance.Total)
 
 	log.Infof("alice claiming onboarding funds...")
-	txid, err := aliceArkClient.Claim(ctx)
+	txid, err := aliceArkClient.SendOffChain(ctx, nil, nil)
```

Comment:
This is bit confusing, in CLI we use SendAsync for covenantless, why do we use SendOffChain in covenantless example? IMO there should be Send and Claim, SendOffChain with receivers=nil, for self send, doesnt seem like best UX.

Reply:
I agree but in my mind the SDK interface should be as minimal as possible. Nothing prevents the developer from creating a more user friendly "Claim" function on top of sendOffchain